certified-connectors/Zoho Calendar/readme.md

```diff
@@ -0,0 +1,61 @@
+# Zoho Calendar
+
+Zoho Calendar connector gives you the power to manage your calendar and perform repetitive actions such as create, edit, delete event by creating workflows using action trigger model.
```

Comment:
Could you please update readme.md file as per this template https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md